security/pkg/pki/util/testdata/3level/certs.sh

```diff
@@ -0,0 +1,39 @@
+# This script generates all keys and certs in the 3level directory.
```

Comment:
Can we rename the directory of this file, i.e., 3level to something more meaningful? And avoid having a leading number '3'.

Reply:
Sure.